sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -119,15 +119,12 @@ static sycl::detail::ESIMDEmuPluginOpaqueData *PiESimdDeviceAccess;
 // Single-entry cache for piPlatformsGet call.
 static pi_platform PiPlatformCache;
 // TODO/FIXME : Memory leak. Handle with 'piTearDown'.
-static sycl::detail::SpinLock *PiPlatformCacheMutex =
-    new sycl::detail::SpinLock;
+static std::mutex *PiPlatformCacheLock = new std::mutex;
```

Comment:
Mustn't it be initialized in init function? (and all other places)

Reply:
Will cover this with other similar variable initialization. (https://github.com/intel/llvm/issues/5599)